index.js

```diff
@@ -61,3 +65,24 @@ export function getToken(token, ...params) {
 
   return params.length === 0 ? langToken : langToken(...params);
 }
+
+export function extend(extendLanguage, opts = {}) {
+  if (!Reflect.has(languages, extendLanguage)) {
+    languages[extendLanguage] = opts;
+
+    return;
+  }
+  Object.assign(languages[extendLanguage], opts);
+}
+
+export function extendFromSystemPath(languagesDirPath) {
+  if (!fs.existsSync(languagesDirPath)) {
+    throw new Error(`The ${languagesDirPath} directory does not exist on this project.`);
+  }
+  const files = fs.readdirSync(languagesDirPath);
```

Comment:
Use `withFileTypes` option to get an Array of dirents. Adding this will allow you to filter between folders and files.